src/integTest/groovy/net/wasdev/wlp/gradle/plugins/TestCompileJSP.groovy

```diff
@@ -0,0 +1,36 @@
+package net.wasdev.wlp.gradle.plugins;
+
+import org.junit.AfterClass
+import org.junit.BeforeClass
+import org.junit.Test
+
+public class TestCompileJSP extends AbstractIntegrationTest{
+    static File resourceDir = new File("build/resources/integrationTest/sample.servlet")
```

Comment:
I don't see any JSP files in sample.servlet.

Reply:
Brought over all the code but forgot to bring the jsp files from the sample project... I'll go ahead and move those over.